_posts/2020-07-11-diversity-for-the-right-solution.md

```diff
@@ -0,0 +1,32 @@
+---
+layout:      posts
+background:  shortBackground
+title:       "Diversity"
+subtitle:    "Don't just build the \"best\" product, build the right one"
+date:        2020-07-11 11:00
+author:      "Kara Waldemer"
+headerImg:  "/assets/images/posts/code.jpg"
+description: "Why diversity is important to successful IT solutions"
+---
+
+## Diversity: Don't just build the "best" product, build the right one
+
+As IT professionals, we make it our goal to always build the best solution that we can. We spend countless hours discussing and designing, building and refactoring, validating and testing. However, despite doing so much to build the best product, we don't always build the *right* product.
+
+For many products, diversity can be the missing key. Diversity in all stages of the pipeline is critical in building not only the “best” solution but, more importantly, the “right” solution. Not sure you agree? Just look at the seat belt.
+
+![Seatbelt](/assets/images/posts/2020-07-10-diversity-for-the-right-solution/alexandria-gilliott-VBFbVw8kZdI-unsplash.jpg){:target="_blank"}
+
+Here is an example of how it is possible to do everything you can think of to build the best product and still get so much wrong. The seat belt has existed for a long time. It has been engineered and reengineered in attempts to improve it and cars go through rigorous testing dedicated to them and other safety features and yet it is still not “right” for many of its users. According to a [study](https://ajph.aphapublications.org/doi/10.2105/AJPH.2011.300275) published in 2011, women are 73% more likely than men to be injured in an accident when wearing a seatbelt.
```

Comment:
Suggest adding `{:target="_blank"}` to the end of the study URL so it opens in a new tab

Reply:
Will do, thanks.